browser/js/content.js

```diff
@@ -288,7 +294,7 @@ function renderSuccess(el, closeCallback, extraHTML) {
 }
 
 // Render an error that replaces the body of the widget.
-function renderErrorFull(el, bodyHTML) {
+function renderErrorFull(el, closeCallback, bodyHTML) {
```

Comment:
Dunno how I feel about a callback being a not-last param but I [set precedent elsewhere](https://github.com/keybase/client/blob/141cf36d4ab1ba8a4bd3230e0fb524e21d2f101d/browser/js/content.js#L138) because I needed to bind a partial. Maybe I should have just made a closure...